app/controllers/curation_concerns/classify_concerns_controller.rb

```diff
@@ -12,13 +11,12 @@ class ClassifyConcernsController < ApplicationController
     helper_method :classify_concern
 
     def new
-      respond_with(classify_concern)
     end
 
     def create
       classify_concern.attributes = params[:classify_concern]
       if classify_concern.valid?
-        respond_with(classify_concern) do |wants|
+        respond_to do |wants|
           wants.html do
             redirect_to new_polymorphic_path(
               [:curation_concerns, classify_concern.curation_concern_class]
```

Comment:
I'm not sure that you would ever submit JSON to that endpoint, but I'll add it for consistency.